internal/pkg/vm/internal/vmcontext/vmcontext.go

```diff
@@ -306,14 +423,16 @@ func (ctx *VMContext) CreateActor(actorID types.Uint64, code cid.Cid, params []i
 	newActor.Code = code
 
 	// send message containing actor's initial balance to construct it with the given params
-	_, _, err = ctx.Runtime().Send(idAddr, types.ConstructorMethodID, ctx.Message().ValueReceived(), params)
-	if err != nil {
-		runtime.Abort("Constructor failed on actor")
-	}
+	ctx.Runtime().Send(idAddr, types.ConstructorMethodID, ctx.Message().ValueReceived(), params)
 
 	return actorAddr
 }
 
+// VerifySignature implemenets the ExtendedInvocationContext interface.
+func (*VMContext) VerifySignature(signer address.Address, signature types.Signature, msg []byte) bool {
```

Comment:
This method appears to be unused. It doesn't seem to belong in this PR. It should either be tested or removed.